src/theme/src/components/table-cell.js

```diff
@@ -0,0 +1,23 @@
+export default function getTableCellStyles(theme) {
+  const { tokens } = theme
+  return {
+    baseStyle: {
+      paddingX: 12,
+      boxSizing: 'border-box',
+      flex: 1,
+      display: 'flex',
+      alignItems: 'center',
+      flexShrink: 0,
+      overflow: 'hidden'
+    },
+    appearances: {
+      default: {
+        _focus: {
+          outline: 'none',
+          background: tokens.colors.blue50,
```

Comment:
Probably worth spitting this out into a `tableCellSelectedBackgroundColor` and `tableCellSelectedBorder` or something similar.

Reply:
I guess it depends on if you want changes to the foundational tokens to impact these component-specific values or not?